ConsoleApp1/Program.cs

```diff
@@ -1,7 +1,16 @@
-﻿namespace ConsoleApp1 {
-    internal class Program {
-        static void Main (string[] args) {
-            Console.WriteLine ("Hello, World!");
-        }
-    }
-}
\ No newline at end of file
+﻿using System;
+namespace MultiplicationTables {
+   class Program {
+      static void Main (string[] args) {
+         for (int i = 1; i <= 10; i++) {
+            Console.WriteLine ($"Multiplication Table for {i}");
+            for (int j = 1; j <= 10; j++) {
+               int result = i * j;
+               Console.WriteLine ($"{i}*{j}={result}");

```

Comment:
No need for local variable 
```suggestion
               Console.WriteLine ($"{i}*{j}={i * j}");

```